src/content/docs/en/pages/devtools/cli/azion-cli/commands/unlink/unlink.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Azion CLI unlink
+description: Unlink an existing project from Azion Edge Computing Platform.
+meta_tags: cli, next, javascript, astro, vue, react, edge computing
+namespace: documentation_cli_unlink
+menu_namespace: cliMenuAlpha
+permalink: /documentation/devtools/cli/unlink-command/
+---
+
+:::note 
+In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
+:::
+
+---
+
+## Unlink
+
+The `azion unlink` command disconnects a local repo or project folder from Azion. The use of the `--yes` option allows the action to complete without asking for confirmation. To get more information about this command, you can use the `-h` or `--help` option.
```

Comment:
```suggestion
## Unlink

The `azion unlink` command disconnects a local repo or project folder from Azion. The use of the `--yes` option allows the action to complete without asking for confirmation. To get more information about this command, you can use the `-h` or `--help` option.

:::note 
In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
:::
```